superset-frontend/src/SqlLab/components/HighlightedSql.jsx

```diff
@@ -41,77 +41,66 @@ const propTypes = {
   shrink: PropTypes.bool,
 };
 
-class HighlightedSql extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      modalBody: null,
-    };
-  }
+function HighlightedSql({ sql, rawSql, maxWidth, maxLines, shrink }) {
```

Comment:
Might be cleaner to move the default properties to the argument list here.

Reply:
Ok, I will get to these tonight. Thank you!